### DISCUSSION TITLE
can't delete nor move e-mails

### DISCUSSION THREAD

**Discussion by Jujubash** [tutao/tutanota#6433] [Bugs]

**Comment by rezbyte** [comment]
Hi @Jujubash!
Sorry to hear you are having issues with your emails. Please email customer support by using the support dialog inside the Tuta Mail client.

To use the support dialog, press 'Show Help' button (![Screenshot from 2024-01-12 12-12-47](https://github.com/tutao/tutanota/assets/48037381/842cb5c0-be02-47cd-85f7-8ef5801a9b4a))on the toolbar on the bottom left corner of the screen, then select 'Support'. The support dialog should appear and then you must write your problem into the text box. After some articles appear, scroll to the very bottom of the list & click the button labelled 'Contact support'.
